sample/QueueTrigger-CSharp/run.csx

```diff
@@ -8,10 +10,4 @@ public static void Run(Message message, out string result, TraceWriter log)
     log.Info($"C# Queue trigger function processed message: {message.Id}");
 
     result = message.Value;
-}
-
-public class Message
```

Comment:
Before these changes, any changes made to Shared\Message.csx would not be picked up (the host wouldn't restart). Now it does.